test/c/test_adjoint.c

```diff
@@ -155,7 +155,7 @@ void test_wav_analysis_adjoint_mw(void **state) {
       dot_complex(f_scal_rec, f_scal, s2let_n_scal(parameters)) -
       dot_complex(f_rec, f, parameters->L * (2 * parameters->L - 1));
 
-  assert_float_equal(creal(error), 0, 1e-12);
```

Comment:
@jasonmcewen, these changes make the tests pass on MacOS, but is it kosher?

Reply:
Thanks @mdavezac.  Instead of completely removing the assert can you just reduce the tolerance?  Instead of 1e-12 perhaps 1e-8 might work?

Reply:
Oh, I didn't see the change, just the removal.  It seems you set to 1e-11 already.  In that case, yes, totally fine to change.